### PR TITLE
Feature/predict hotfix

### PR DESCRIPTION
**Pull request by JanFidor**

### CODE REVIEW

**Review by Kieru18 — APPROVED** [review]
looks fine 👍

File: e_motion/ml/coco/coco_utils.py [hunk 1]
```diff
@@ -197,11 +197,19 @@ def __getitem__(self, idx):
         return img, target
 
 
-def get_coco(root, image_set, transforms, mode="instances", use_v2=False, with_masks=False):
+def get_coco(
```
Comment (WojciechL02) [inline e_motion/ml/coco/coco_utils.py:200]: super-nice linter

**Review by WojciechL02 — APPROVED** [review]
good practices

**Review by ziarekk — APPROVED** [review]
Looks good

**Pull request merged**